lib/PuppeteerSharp/Helpers/RemoteObjectHelper.cs

```diff
@@ -92,15 +94,15 @@ private static object ValueFromType<T>(JToken value, RemoteObjectType objectType
                 switch (objectType)
                 {
                     case RemoteObjectType.Object:
-                        return value.ToObject<T>(true);
+                        return value.ToObject<T>();
                     case RemoteObjectType.Undefined:
                         return null;
                     case RemoteObjectType.Number:
-                        return value.Value<T>();
+                        return value.GetInt32();
                     case RemoteObjectType.Boolean:
-                        return value.Value<bool>();
+                        return value.GetBoolean();
                     case RemoteObjectType.Bigint:
-                        return value.Value<double>();
+                        return value.GetInt64();
```

Comment:
Just double checking that `Number` and `Bigint` should now be mapped to `Int32` and `Int64`, respectively

Reply:
Good one. I restored `Bigint` to `GetDouble` and `Number` to `ToObject<T>`.